### PR TITLE
Squash content of Marco Polo into one commit

### PR DESCRIPTION
**Pull request by Honkou**
commit 10bd24c545a0288894ccbce8678a74be79726efd
Author: Honkou <filiphancke@gmail.com>
Date:   Sat May 13 22:16:32 2023 +0200

    Ignore vscode settings

commit 839f7c5ba7349899d00732e49fb058895a353986
Author: Honkou <filiphancke@gmail.com>
Date:   Sat May 13 22:16:05 2023 +0200

    Rewrite core using good practices

### CODE REVIEW

File: source/handle_messages.py
Comment (MariaDabrowska) [inline source/handle_messages.py:1]: There is code that needs tests, at least Marco-Polo part. Please fix ;)

File: source/main.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import os
+
+import discord
+from discord.ext import commands
+from dotenv import load_dotenv
+from handle_messages import Listeners
+
+
+def get_token() -> str:
+    """Find and return proper authorization token."""
```
Comment (Szybki1000) [inline source/main.py]:
```suggestion
    """Find and return proper Discord authorization token."""
```

**Review by MariaDabrowska — DISMISSED** [review]

File: source/handle_messages.py
Comment (Honkou) [reply source/handle_messages.py:1]: Will do in a separate PR, as part of adding tests to CI.

File: source/main.py (hunk 1 above)
Comment (Honkou) [reply source/main.py]: Good point

**Review by MariaDabrowska — APPROVED** [review]

**Pull request merged**